internal/rdc/client.go

```diff
@@ -152,12 +155,18 @@ func (c *Client) PollJob(ctx context.Context, id string, interval, timeout time.
 	deathclock := time.NewTimer(timeout)
 	defer deathclock.Stop()
 
+	var retryTime int
 	for {
```

Comment:
The ticket mentions that the logging needs to be improved as well. Because when this status poll fails right now, we have no idea due to what reason (nothing is logged aside from the fact that it failed). Was it network? Was it an unexpected status code or payload from the server? Which makes it impossible to troubleshoot any further.

Reply:
I've added some logs for retrying.